py/vm.c

```diff
@@ -927,8 +927,8 @@ unwind_jump:;
                     // unum & 0xff == n_positional
                     // (unum >> 8) & 0xff == n_keyword
                     // We have following stack layout here:
-                    // fun arg0 arg1 ... kw0 val0 kw1 val1 ... seq dict <- TOS
-                    sp -= (unum & 0xff) + ((unum >> 7) & 0x1fe) + 2;
+                    // fun arg0 arg1 ... kw0 val0 kw1 val1 ... bitmap <- TOS
```

Comment:
In this commit the `bitmap` does not exist so it should be `seq` instead.  I.e., this commit should just delete the `dict` entry.  Then in the next commit, `seq` is renamed to `bitmap`.

Reply:
done